cmd/kk/pkg/bootstrap/customscripts/module.go

```diff
@@ -49,4 +49,24 @@ func (m *CustomScriptsModule) Init() {
 
 		m.Tasks = append(m.Tasks, task)
 	}
+
+	m.CopyKubeConfig()
+}
+
+func (m *CustomScriptsModule) CopyKubeConfig() {
+	taskDir := fmt.Sprintf("%s-default-script", m.Phase)
+	taskName := fmt.Sprintf("Phase:%s CopyKubeConfig", m.Phase)
+	script := kubekeyapiv1alpha2.CustomScripts{
+		Name: "CopyKubeConfig",
+		Bash: "cp /etc/kubernetes/admin.conf $HOME/.kube/config",
```

Comment:
you need double check the permission and owner for admin.conf.  kubekey requirements for root user to install. if you use normal user, then the design will broken and introduce many more issues abt issues.

Reply:
All official documents use `ubuntu` users... https://github.com/kubesphere/kubekey/blob/master/docs/config-example.md

Is it ok to use `sudo cp` here? Or maybe I'll test it again after a while.